### PR TITLE
Common lib

### PR DESCRIPTION
**Pull request by Jlrine2**

### CODE REVIEW

File: apps/api/requirements-api.txt [hunk 1]
```diff
@@ -8,3 +8,4 @@ requests
 serverless_wsgi>=1.7.8
 shapely
 strict-rfc3339
+-e lib/dynamo
```
Comment (jhkennedy) [inline apps/api/requirements-api.txt:11]: I don't actually see this folder/library in the PR, did you push it?

Comment (Jlrine2) [reply apps/api/requirements-api.txt:11]: I did not

File: lib/dynamo/setup.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+setup(
+    name='dynamo',
+
+    license='BSD',
+    include_package_data=True,
+
+    python_requires='~=3.8',
+
+    packages=find_packages(),
+)
```
Comment (jhkennedy) [inline lib/dynamo/setup.py:12]: Probably should have a `install_requires=`

**Review by jhkennedy — APPROVED** [review]
This makes sense to me. `lib/dynamo/dynamo` implies more libraries are coming, which I think is probably a good decision.

**Pull request merged**